docs/en_US/Compressor/QuickStart.md

```diff
@@ -1,8 +1,12 @@
-# Quick Start to Compress a Model
+# Tutorial for Model Compression
+
+In this tutorial, we use the [first section](#quick-start-to-compress-a-model) to quickly go through the usage of model compression on NNI. Then use the [second section](#detailed-usage-guide) to explain more details of the usage.
+
+## Quick Start to Compress a Model
 
 NNI provides very simple APIs for compressing a model. The compression includes pruning algorithms and quantization algorithms. The usage of them are the same, thus, here we use slim pruner as an example to show the usage.
 
```

Comment:
suggest adding a link for [slim pruner](https://github.com/microsoft/nni/blob/master/docs/en_US/Compressor/Pruner.md#slim-pruner)

Reply:
good point, thanks!